pkg/controller/import-controller.go

```diff
@@ -72,6 +93,56 @@ func (ic *ImportController) findImportPodFromCache(pvc *v1.PersistentVolumeClaim
 	return podList[0], nil
 }
 
+func (ic *ImportController) importerRequiresScratchSpace(pvc *v1.PersistentVolumeClaim) bool {
```

Comment:
I agree the name is a little too long and verbose (doesn't need importer, since we are in the importer controller), but I don't think that a 'requiresX' is less clear than shouldUse, requires is much more explicit about the actual hard need, vs shouldUse is more of a yeah we should but we can probably get away with not doing whatever it is.